bot/extensions/wikipedia_cog.py

```diff
@@ -1,26 +1,40 @@
 from discord.ext.commands import Cog, hybrid_command
 from discord.ui import View
-from discord import ButtonStyle, ui
+from discord import ButtonStyle, ui, Embed
 from urllib.request import urlopen
 from urllib.parse import quote_plus
 from json import loads
-from discord import Embed
 
 
-def search_results(search):
+def search_results(search: str):
```

Comment:
You forgot the return type hint here.  I think it's a list of any (`-> List[Any]`)

Reply:
ahhh!!!!